### ISSUE TITLE
color flag is missing

### ISSUE THREAD

**Issue by jimsmith** [infracost/infracost#507]
Hey, I don't see how to get color by default as it's set to 'no-color'

`time="2021-03-10T17:00:31Z" level=info msg="Running command: /usr/bin/terraform plan -input=false -lock=false -no-color -var-file=./../../variables/dev.tfvars -out=/tmp/tfplan508083620"`

`time="2021-03-10T17:00:39Z" level=info msg="Running command: /usr/bin/terraform show -no-color -json /tmp/tfplan508083620"`

I don't see a `--color` flag 

```
docker run --rm -v $PWD/:/code/ infracost/infracost breakdown --help | grep color
      --no-color           Turn off colored output
```

**Comment by aliscott** [comment]
@jimsmith thanks for the issue. Currently the Infracost output itself color by default and the `--no-color` flag disables that.

Is it the output from the terraform binary you are wanting the color option for? When we run the terraform binary under the hood we always run them with no-color since the output goes to the logs.

**Comment by jimsmith** [comment]
@aliscott good point, am I missing something obvious 👀 or something else as it's the latest docker version using `breakdown`  as I see `-no-color` by default

```
	INFRACOST_API_KEY=
	LOG_LEVEL=INFO

	docker run --rm \
	-e INFRACOST_API_KEY=$INFRACOST_API_KEY \
	-e AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY_ID \
	-e AWS_SECRET_ACCESS_KEY=$AWS_SECRET_ACCESS_KEY \
	-e AWS_SESSION_TOKEN=$AWS_SESSION_TOKEN \
	-e AWS_REGION=$AWS_REGION \
	-v $PWD/:/code/ infracost/infracost breakdown --path /code/infrastructure/aws/environments/dev/ \
	--terraform-plan-flags="-var-file=./../../variables/dev.tfvars \
	--log-level $LOG_LEVEL
```

```
time="2021-03-11T09:01:17Z" level=info msg="Detected Terraform directory at /code/infrastructure/aws/environments/dev/"
time="2021-03-11T09:01:18Z" level=info msg="starting: Running terraform plan"
time="2021-03-11T09:01:18Z" level=info msg="Running command: /usr/bin/terraform plan -input=false -lock=false -no-color -var-file=./../../variables/dev.tfvars -out=/tmp/tfplan505386012"
time="2021-03-11T09:01:25Z" level=info msg="starting: Running terraform show"
time="2021-03-11T09:01:25Z" level=info msg="Running command: /usr/bin/terraform show -no-color -json /tmp/tfplan505386012"
time="2021-03-11T09:01:29Z" level=info msg="starting: Calculating monthly cost estimate"
```

Then the infracost output.... just white and grey text

**Comment by aliscott** [comment]
Thanks @jimsmith, good spot! Yeah the `--log-level` disables the spinners and logs those steps instead - spinners don't play nicely with logs. It still keeps the coloured output in the breakdown table (although this is only white and grey), the diff (this has more colors) and any errors. I've updated the docs to clarify that log-level disables the spinners.

Closing this just now - let me know if you find any more issues and I will reopen it.
